tracing-attributes/src/attr.rs

```diff
@@ -282,7 +282,7 @@ impl Parse for FormatMode {
             match ident.to_string().as_str() {
                 "Debug" => Ok(FormatMode::Debug),
                 "Display" => Ok(FormatMode::Display),
-                "Raw" => Ok(FormatMode::Raw),
+                "Raw" => Ok(FormatMode::Value),
```

Comment:
this should also be renamed to `Value`:
```suggestion
                "Value" => Ok(FormatMode::Value),
```